test/rmq/test_process_comms.py

```diff
@@ -194,6 +194,7 @@ def test_kill(self):
 
         # Check the outcome
         self.assertTrue(result)
+        # Occasionally fail
```

Comment:
Do you mean this test occasionally fails? If yes, probably better write `The following check occasionally fails`.

Even better would be of course to figure out what is causing this...

Reply:
Ah, sorry I just noticed https://github.com/aiidateam/plumpy/issues/109

Reply:
Yes, it is the failed test. I have tried to figure this out, but it is difficult to reproduce.(Even I use pytest-repeat `py.test --cache-clear --count=3000`)